deploy/catalog/templates/apiserver.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: apiserver-ssl
           mountPath: /var/run/kubernetes-service-catalog
           readOnly: true
-      {{ if eq (default .Values.storageType "etcd") "etcd" -}}
+      {{ if eq (.Values.storageType | default "etcd") "etcd" -}}
```

Comment:
@krancour looks like we had the default values order mixed up? I thought the one that got removed was right, but anyway we can just wait for your chart changes. Thanks for the reviews!